Marlin/src/MarlinCore.cpp

```diff
@@ -533,6 +533,20 @@ inline void manage_inactivity(const bool ignore_stepper_queue=false) {
     }
   #endif
 
+  #if HAS_FILAMENTCHANGE
+    // Handle a standalone FILAMENTCHANGE button
+    constexpr millis_t FILAMENTCHANGE_DEBOUNCE_DELAY = 1000UL;
+    static millis_t next_filamentchange_key_ms; // = 0
+    if !READ(FILAMENTCHANGE_PIN)) { // FILAMENTCHANGE_PIN goes LOW when pressed
```

Comment:
In C++ the if keyword requires parentheses around the conditions.

Reply:
parentheses around the conditions